pkg/util/utils.go

```diff
@@ -220,9 +223,59 @@ func UploadError(b []byte, url string) error {
 	return nil
 }
 
-func MaybeReportErrorAndExit(err error) {
+func MaybeReportErrorAndExit(errToReport error) {
+	var err error
 	if viper.GetBool(config.WantReportError) {
-		ReportError(err, reportingURL)
+		err = ReportError(errToReport, reportingURL)
+		if err != nil {
+			glog.Errorf(err.Error())
+		}
+	} else {
+		fmt.Println("An error has occurred.  Would you like to opt in to sending this and future crashes to minikube to help prevent future errors?")
+		if PromptUserForAccept(os.Stdin) {
+			mConf, err := minikubeConfig.ReadConfig()
+			if err != nil {
+				glog.Errorf(err.Error())
+			}
+			minikubeConfig.SetBool(mConf, config.WantReportError, "true")
```

Comment:
instead of calling this directly, can you add it to the map at https://github.com/kubernetes/minikube/blob/master/cmd/minikube/cmd/config/config.go

and then use the Set command

then we can add callbacks and this can be configurable through `minikube config set`